qemu/tests/qemu_img.py

```diff
@@ -177,6 +178,45 @@ def create_test(cmd):
                 cluster_size=params.get("image_cluster_size"))
         remove(img)
 
+    def send_signal(timeout=360):
+        """
+        send signal "SIGUSR1" to qemu-img without the option -p
+        to report progress
+        """
+        logging.info("Send signal to qemu-img")
+        end_time = time.time() + timeout
+        while time.time() < end_time:
+            time.sleep(1)
+            status = utils.system("kill -SIGUSR1 `pidof qemu-img`",
+                                  ignore_status=True)
+            if status == 0:
+                return
+        logging.info("Fail to get pid of qemu-img")
+
+    def check_command_output(CmdResult):
+        """
+        Check standard error or standard output of command
+        : param CmdResult: a list of CmdResult objects
+        """
+        logging.info("Check result of command")
+        check_output = params.get("check_output", "")
+        if not check_output:
+            if CmdResult.exit_status == 0:
+                return
+            else:
+                err_msg = "Get nonzero exit status '%s'" % CmdResult.command
+                raise error.TestFail(err_msg)
+        if check_output == "stderr":
```

Comment:
how about   
if not hasattr(CmdResult, check_output):
     output = ""
else:
     output = getattr(CmdResult, check_output)